docs/getting_started/overview.rst

```diff
@@ -0,0 +1,53 @@
+************
+Overview
+************
+
+The ``trifinger_simulation`` package provides an interface to the TriFinger robots
+in simulation using the `pybullet physics engine <https://pypi.org/project/pybullet/>`_.
+
+.. note::
```

Comment:
This "note" is big enough that it could be it's own section instead (something like "About the TriFinger Robots").

Reply:
I think this is way too much content for a footnote (especially as it contains figures).  I really think it would make sense as a proper section.